<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>targetFinder</class>
 <widget class="QDialog" name="targetFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_broadcast">
       <property name="text">
        <string>BroadCast address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_broadcast">
       <property name="text">
        <string>192.168.2.255</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button_find">
     <property name="text">
      <string>Find target</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_reset">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Reset target</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
